src/components/sidebar/index.tsx

```diff
@@ -45,6 +48,7 @@ const Sidebar = ({
 	visuallyHideTitle = false,
 	isInstallPage,
 	showResourcesList = true,
+	className,
 }: SidebarProps): ReactElement => {
```

Comment:
If you add this here, you can remove the [className prop](https://github.com/hashicorp/dev-portal/pull/2568/files#diff-88d4455b7e732cfa9c1f1184c73c936454674a54824e703fce03775a8bceaf20R197-R200) from `src/components/sidebar/types.ts`
```suggestion
}: SidebarProps & ) & JSX.IntrinsicElements['div'] => {
```

Reply:
Thanks for the suggestion! I managed to get this going by changing the line to:

```ts
}: SidebarProps & JSX.IntrinsicElements['div']) => {
```

The commit:
52734ef